frontend/src/toolbar/button/toolbarButtonLogic.ts

```diff
@@ -162,5 +163,14 @@ export const toolbarButtonLogic = kea<toolbarButtonLogicType>({
         hideActionsInfo: () => {
             actionsTabLogic.actions.selectAction(null)
         },
+        showStats: () => {
+            posthog.capture('toolbar mode triggered', { mode: 'stats', enabled: true })
+        },
+        hideStats: () => {
+            posthog.capture('toolbar mode triggered', { mode: 'stats', enabled: false })
+        },
+        setExtensionPercentage: ({ percentage }) => {
+            posthog.capture('toolbar set extension percentage', { percentage })
+        },
```

Comment:
This can be ignored. It'll be called a thousand times per page, as it sets the percentage the buttons are extended from the middle. 0% means they're behind the logo, 100% means they're fully out. 

![2021-01-06 12 10 25](https://user-images.githubusercontent.com/53387/103762232-36936a80-5018-11eb-8130-740aa031777b.gif)

Reply:
Good point, I've switched this in favor of `toolbar loaded`

Reply:
Removed in favor of `toolbar loaded`